### PR TITLE
HH-142261 review docs

### PR DESCRIPTION
**Pull request by nicholasgribanov**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -16,36 +16,44 @@ These options can be set for each Frontik instance (see [options.py](/frontik/op
 | `app`                        | `str`   | `None`        | Application name  (is used for application identification and as default path to it's class)|
 | `app_class`                  | `str`   | `None`        | Path to application class (by default `FrontikApplication` class is used (see [Frontik application structure](/docs/frontik-app.md))) |
 | `workers`                    | `int`   | `1`           | Number of worker processes creates using fork. When default value is used, master itself become worker, without fork |
+| `init_workers_timeout_sec`   | `int`   | `60`          | Max time for initialization worker                                     |
```
Comment (Aulust) [inline docs/config.md:19]: Timeout for worker initialization?

File: docs/config.md [hunk 2]
```diff
@@ -16,36 +16,44 @@ These options can be set for each Frontik instance (see [options.py](/frontik/op
...
 | `xheaders  `                 | `bool`  | `False`       | Controls Tornado HTTPServer `xheaders` option                          |
 | `tornado_settings`           | `dict`  | `None`        | tornado.web.Application settings                                       |
 | `autoreload`                 | `bool`  | `False`       | Restart Frontik after changes in application sources or config files   |
 | `debug`                      | `bool`  | `False`       | Enable debug mode                                                      |
 | `debug_login`                | `str`   | `None`        | Debug mode login for basic authentication (when `debug=False`)         |
 | `debug_password`             | `str`   | `None`        | Debug mode password for basic authentication (when `debug=False`)      |
-| `handlers_count`             | `int`   | `100`         | Limit for number of simultaneous requests handled by Frontik instance  |
+| `max_active_handlers`        | `int`   | `100`         | Limit for number of simultaneous requests handled by Frontik instance  |
```
Comment (Aulust) [inline docs/config.md:26]: Limit the maximum number of simultaneous requests...

File: docs/config.md [hunk 3]
```diff
@@ -16,36 +16,44 @@ These options can be set for each Frontik instance (see [options.py](/frontik/op
...
 | `datacenter`                 | `str`   | `None`        | Datacenter where current application is running                        |
+| `validate_request_id`        | `bool`  | `False`       | Enable check for valid format request_id (32 characters hex-string)    |
+| `node_name`                  | `str`   | `False`       | Name of node where startup frontik application                         |
```
Comment (pvorlov) [inline docs/config.md:29]: Name of node where frontik application starts.

File: docs/config.md [hunk 4]
```diff
@@ -16,36 +16,44 @@ These options can be set for each Frontik instance (see [options.py](/frontik/op
...
 | `tornado_settings`           | `dict`  | `None`        | tornado.web.Application settings                                       |
 | `autoreload`                 | `bool`  | `False`       | Restart Frontik after changes in application sources or config files   |
 | `debug`                      | `bool`  | `False`       | Enable debug mode                                                      |
 | `debug_login`                | `str`   | `None`        | Debug mode login for basic authentication (when `debug=False`)         |
 | `debug_password`             | `str`   | `None`        | Debug mode password for basic authentication (when `debug=False`)      |
-| `handlers_count`             | `int`   | `100`         | Limit for number of simultaneous requests handled by Frontik instance  |
+| `max_active_handlers`        | `int`   | `100`         | Limit for number of simultaneous requests handled by Frontik instance  |
 | `datacenter`                 | `str`   | `None`        | Datacenter where current application is running                        |
+| `validate_request_id`        | `bool`  | `False`       | Enable check for valid format request_id (32 characters hex-string)    |
```
Comment (Aulust) [inline docs/config.md:28]: for validity of request_id (32 characters hex-string)

File: docs/config.md [hunk 5]
```diff
@@ -16,36 +16,44 @@ These options can be set for each Frontik instance (see [options.py](/frontik/op
...
+| `node_name`                  | `str`   | `False`       | Name of node where startup frontik application                         |
+| `common_executor_pool_size`  | `int`   | `10`          | The maximum number of threads that can be used to execute the given calls |
+| `stop_timeout`               | `int`   | `3`           | Max time in seconds for gracefully stopping application                |
```
Comment (pvorlov) [inline docs/config.md:31]: Max time in seconds for gracefully application stopping

File: docs/config.md [hunk 6]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
 
+Consul options:
+
+| Option name                                 | Type   | Default value | Description                                                            |
+|---------------------------------------------|--------|---------------|------------------------------------------------------------------------|
+| `consul_enabled`                            | `bool` | `True`        | Enable Consul features: registration, kv-store, upstreams              |
+| `consul_host`                               | `str`  | `127.0.0.1`   | Consul host                                                            |
+| `consul_port`                               | `int`  | `None`        | Consul port                                                            |    
+| `consul_service_address`                    | `str`  | `None`        | Address of application for Consul registration                         |
+| `consul_check_host`                         | `str`  | `None`        | Address for healthcheck application, use `consul_service_address` if not given  |
```
Comment (Aulust) [inline docs/config.md:99]: ,if not provided, `consul_service_address` is used

File: docs/config.md [hunk 7]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_http_check_interval_sec`            | `int`  | `10`          | Interval for getting healthcheck request                               |
```
Comment (Aulust) [inline docs/config.md:100]: getting -> making

File: docs/config.md [hunk 8]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_http_check_timeout_sec`             | `int`  | `1`           | Timeout for getting healthcheck request                                |
```
Comment (Aulust) [inline docs/config.md:101]: Timeout for health check request

File: docs/config.md [hunk 9]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_tags`                               | `list` | `[]`          | Additional parameters, which will sent with Consul registration        |
```
Comment (Aulust) [inline docs/config.md:102]: will be sent или to send

File: docs/config.md [hunk 10]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_cache_initial_warmup_timeout_sec`   | `int`  | `2`           | Ttl of HTTP session for initialize cache                               |
+| `consul_cache_backoff_delay_seconds`        | `int`  | `10`          | Timeout for trying to get some info from Consul after some Exception   |
+| `consul_consistency_mode`                   | `str`  | `default`     | May set as `default`, `consistent` and `stale`. See Consul documentation   |
+| `consul_weight_consistency_mode`            | `str`  | `default`     | Consistency mode for get weigth.                                       |
+| `consul_deregister_critical_timeout`        | `str`  | `120h`        | Time, after that service will be de-registered automatically from Consul |
+| `upstreams`                                 | `list` | `[]`          | List of service upstreams - services, where current app will send http requests |
+| `fail_start_on_empty_upstream`              | `bool` | `True`        | If `True` app will not running, if one or more upstreams not have hosts addresses |
```
Comment (Aulust) [inline docs/config.md:111]: app will not start if one or more upstreams don't have host addresses

File: docs/config.md [hunk 11]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_check_host`                         | `str`  | `None`        | Address for healthcheck application, use `consul_service_address` if not given  |
+| `consul_http_check_interval_sec`            | `int`  | `10`          | Interval for getting healthcheck request                               |
+| `consul_http_check_timeout_sec`             | `int`  | `1`           | Timeout for getting healthcheck request                                |
+| `consul_tags`                               | `list` | `[]`          | Additional parameters, which will sent with Consul registration        |
+| `consul_weight_watch_seconds`               | `int`  | `600`         | Max waiting blocking query time for get host weight from Consul        |
+| `consul_weight_total_timeout_sec`           | `int`  | `650`         | Timeout for waiting blocking query for getting host weight from Consul |
+| `consul_cache_initial_warmup_timeout_sec`   | `int`  | `2`           | Ttl of HTTP session for initialize cache                               |
+| `consul_cache_backoff_delay_seconds`        | `int`  | `10`          | Timeout for trying to get some info from Consul after some Exception   |
```
Comment (Aulust) [inline docs/config.md:106]: Может быть Interval to wait until next attempt to fetch info from consul after exception

File: docs/config.md [hunk 12]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_service_address`                    | `str`  | `None`        | Address of application for Consul registration                         |
+| `consul_check_host`                         | `str`  | `None`        | Address for healthcheck application, use `consul_service_address` if not given  |
+| `consul_http_check_interval_sec`            | `int`  | `10`          | Interval for getting healthcheck request                               |
+| `consul_http_check_timeout_sec`             | `int`  | `1`           | Timeout for getting healthcheck request                                |
+| `consul_tags`                               | `list` | `[]`          | Additional parameters, which will sent with Consul registration        |
+| `consul_weight_watch_seconds`               | `int`  | `600`         | Max waiting blocking query time for get host weight from Consul        |
+| `consul_weight_total_timeout_sec`           | `int`  | `650`         | Timeout for waiting blocking query for getting host weight from Consul |
+| `consul_cache_initial_warmup_timeout_sec`   | `int`  | `2`           | Ttl of HTTP session for initialize cache                               |
```
Comment (Aulust) [inline docs/config.md:105]: for -> to

File: docs/config.md [hunk 13]
```diff
@@ -79,6 +88,37 @@ Producers options:
...
+| `consul_weight_consistency_mode`            | `str`  | `default`     | Consistency mode for get weigth.                                       |
+| `consul_deregister_critical_timeout`        | `str`  | `120h`        | Time, after that service will be de-registered automatically from Consul |
+| `upstreams`                                 | `list` | `[]`          | List of service upstreams - services, where current app will send http requests |
+| `fail_start_on_empty_upstream`              | `bool` | `True`        | If `True` app will not running, if one or more upstreams not have hosts addresses |
+
+Opentelemetry options:
+
+| Option name                         | Type   | Default value | Description                                                            |
+|-------------------------------------|--------|---------------|------------------------------------------------------------------------|
+| `opentelemetry_collector_host`      | `str`  | `127.0.0.1`   | Opentelemetry collector host                                           |
```
Comment (pvorlov) [inline docs/config.md:117]: Opentelemetry -> OpenTelemetry?

**Review by Aulust — APPROVED** [review]

**Review by pvorlov — APPROVED** [review]

**Pull request merged**